### DISCUSSION TITLE
关于鉴别和剔除虚假数据的讨论

### DISCUSSION THREAD

**Discussion by lilongsky** [CollegesChat/university-information#81] [Enhancement]
在翻查 [Github的学校页面](https://github.com/CollegesChat/university-information/tree/master/questionnaires)时，发现一个明显为虚假的学校数据。
 链接：
`https://github.com/CollegesChat/university-information/blob/master/questionnaires/universities/%E9%BA%BB%E7%9C%81%E7%90%86%E5%B7%A5.md`

条件允许的话，可以考虑招募志愿者对于问卷进行一轮检视，排除那些明显的虚假数据。
或者在允许在讨论区集中反馈后修改？

**Comment by slanterns** [comment]
（还有 [仲恺说唱职业技术学院](https://github.com/CollegesChat/university-information/blob/master/questionnaires/universities/%E4%BB%B2%E6%81%BA%E8%AF%B4%E5%94%B1%E8%81%8C%E4%B8%9A%E6%8A%80%E6%9C%AF%E5%AD%A6%E9%99%A2.md)

**Reply by YanWQ-monad** [reply]
还有[家里蹲大学](https://github.com/CollegesChat/university-information/blob/master/questionnaires/universities/%E5%AE%B6%E9%87%8C%E8%B9%B2%E5%A4%A7%E5%AD%A6.md)呢（笑

**Reply by lilongsky** [reply]
刚刚还看到了一个[喵内噶大学](https://github.com/CollegesChat/university-information/blob/master/questionnaires/universities/%E5%96%B5%E5%86%85%E5%99%B6%E5%A4%A7%E5%AD%A6.md) (笑

**Reply by nexmoe** [reply]
这种名称的可以写个脚本对照百度百科自动处理

**Comment by YanWQ-monad** [comment]
关于这个滥用、虚假的数据

如果先每个都查一遍再放出来，工作量太大了，问卷几乎是一天能有 1k 份的量。

现在我们的策略是：
只会做很基础的 review（可能甚至不会），确认信息是有效的就会放上去
**abuse 之类的都需要大家在看的过程中提出来**
目前的人手只能做到这样了

对于集中反馈帖这个正有想法，应该不久后可以有。

**Reply by aisuneko** [reply]
我的建议是写个关键词过滤脚本初审一遍  省下部分workload

**Comment by ghost** [comment]
一个粗暴的建议，如果能按照ip排列的话，遇到虚假数据就将那个ip段所有的提交拎出来放到最后处理。
